res/database/Default/user/cujo.xml

```diff
@@ -0,0 +1,1807 @@
+        <news id="cujo-news-internet-trend-01" type="0" thread_id="cujo-news-internet-trend" creator="8827" created_by="Cujo">
+            <title>
+                <de>Willy Bates: Internet ist nur ein Trend</de>
+                <en>Willy Bates: Internet is just a trend</en>
+            </title>
+            <description>
+                <de>In einem Fernseh-Interview prophezeit der Software-Entwickler Willy Bates dem Internet keine große Zukunft: "Das ist nur ein kurzfristiger Trend. In 10 Jahren gibt es das nicht mehr."</de>
+                <en>In a television interview, software developer Willy Bates doesn't predict a great future for the Internet: "It's just a short-term trend. It won't be around in 10 years."</en>
+            </description>
+            <data genre="3" price="0.8" quality="37" />
+            <availability year_range_from="1993" year_range_to="1995" />
+        </news>
+		<!-- "Wildecker Herzbuben" -->
+        <news id="cujo-news-wildberger-karobuben-01" type="0" thread_id="cujo-news-wildberger-karobuben" creator="8827" created_by="Cujo">
+            <title>
+                <de>Wildberger Karobuben: "Wir sind jetzt schlank"</de>
+                <en>Wildberger Karobuben: "We are now slim"</en>
+            </title>
+            <description>
+                <de>In einem Exklusiv-Interview mit der Zeitschrift FARBIGE erzählen die Wildberger Karobuben von ihrer Hungerkur: "Wir haben 7 Monate lang fast nichts gegessen und in dieser Zeit 300 Gramm abgenommen. Zusammengerechnet natürlich."</de>
+                <en>In an exclusive interview with FARBIGE magazine, the Wildberg Karobuben talk about their starvation diet: "We ate almost nothing for 7 months and lost 300 grams in that time. Added up, of course."</en>
+            </description>
+            <data genre="1" price="1.03" quality="19" />
+            <availability year_range_from="1989" year_range_to="" />
+        </news>
+		<!-- "Stiftung Warentest" -->		
+        <news id="cujo-news-stiftung-guetertest-01" type="0" thread_id="cujo-news-stiftung-guetertest" creator="8827" created_by="Cujo">
+            <title>
+                <de>Stiftung Gütertest testet Kugelschreiber</de>
+                <en>Fondation goodstest tests ballpoint pens</en>
+            </title>
+            <description>
+                <de>Die Stiftung Gütertest hat 5.748 Kugelschreiber getestet. Den besten Stift stellte die Firma Bastilo her. Erst nach einer Schreibdauer von 723 Stunden war seine Mine leer. 7 Tester wurden wegen eines Schreibkrampfes stationär behandelt.</de>
+                <en>Fondation goodstest tested 5,748 ballpoint pens. The best ballpoint pen was produced by the Bastilo company. Its refill was only empty after writing for 723 hours. 7 testers were hospitalized due to writer's cramp.</en>
+            </description>
+            <effects>
+                <effect trigger="happen" type="triggernews" time="2,2,5,17,21" news="cujo-news-stiftung-guetertest-02" />

```

Comment:
Zusammenhang nicht klar. Vielleicht lassen sich hier auch Variablen nutzen (eine Nachricht, unterschiedliche Dinge getestet. Die aktuelle Variablenlogik erlaubt das. Beispiel ist bei Ronny in de Kultur-Skripten zu finden.

Reply:
Muss bei einer Newskette ein Zusammenhang bestehen? Mir ging es darum, dass zwischen den Gütertest-News 2-5 Tage Abstand sind.

Wenn es eine andere Möglichkeit dies zu erreichen, mache ich das gerne.

Wenn es nur 2 News zu einem Thema gibt, sollten die nicht direkt aufeinander folgen. Sollte es irgendwann vielleicht mal 10 Gütertest-News geben, kann man den Abstand auch auf 1-2 Tage verkürzen.

Reply:
Newsketten brauchen keinen direkten Zusammenhang. Allerdings kann man "Kettenglieder" so definieren, dass sie nicht von selbst "passieren" koennen (keine Initialnews).

Auch koennen Nachrichten andere Nachrichtenketten "ausloesen".

Reply:
Ich weiß nicht, ob ich mich diesbezüglich kurz und klar verständlich ausdrücken kann. Man erreicht mit diesem Pattern zwar, dass die Nachrichten nicht gleichzeitig kommen. Man erreicht aber auch, dass sobald die erste Nachricht kommt, die anderen IMMER auch kommen. Und es gibt zwischen diesen beiden Tests (oder den verschiedenen TVTower-Nachrichten) keinen Zusammenhang, insb. keinen zeitlichen.

Nachfolgetrigger würde ich gerne nur verwenden, wenn es einen Zusammenhang gibt (sich der Zeitpunkt der Nachfolgenachricht rechtfertigen lässt). Der könnte z.B. auch darin bestehen, dass es eine (nicht in der GUI sichtbare) Triggernachricht gibt, die dann zufällig eine Nachricht anstößt. Aktuell unterstützt sind (glaube ich) nur vier Alternativen, was die ganze Sache syntaktisch einschränkt (eine von vier Nachrichten wird mit 25% Wahrscheinlichkeit ausgelöst).

Ich könnte mir technisch aber folgenden Ansatz vorstellen. Es gibt beliebig viele Erstnachrichten, die zum selben Thread gehören. Und beim Auswürfeln wird darauf geachtet, dass gleichzeitig möglichst keine Erstnachrichten aus demselben Thread auftauchen (und die Wahrscheinlichkeit gering ist, wenn der Thread gerade erst dran war).
Es ist eher eine Frameworkaufgabe, Wiederholungen zu reduzieren. Das sollte nicht mit "Workarounds" in der Datenbankdefinition erfolgen.